plugins/rplanners/rampoptimizer/ramp.cpp

```diff
@@ -989,7 +989,7 @@ void ParabolicPath::FindRampNDIndex(dReal t, int& index, dReal& remainder) const
         int index_ = 0;
         dReal curTime = 0;
         std::vector<RampND>::const_iterator itrampnd = _rampnds.begin();
-        while ((itrampnd - 1) != _rampnds.end() && t >= curTime) {
+        while( itrampnd != _rampnds.end() && t >= curTime ) {
```

Comment:
There is similar code in ``ParabolicCurve::FindRampIndex``, please check that as well.

Reply:
@kanbouchou that's true. fixed `PiecewisePolynomial::FindPolynomialIndex` as well. could you review them?

Reply:
@Puttichai  can you check?

Reply:
Looks good to me.

Reply:
looks good to me too